GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -245,6 +257,14 @@ @interface GDTCORApplication ()
 
 @implementation GDTCORApplication
 
+/** A dispatch queue on which all task semaphore will populate and remove from
+ * gBackgroundIdentifierToSemaphoreMap.
+ */
+static dispatch_queue_t gSemaphoreQueue;
+
+/** For mapping backgroundIdentifier to task semaphore. */
+static NSMutableDictionary<NSNumber *, dispatch_semaphore_t> *gBackgroundIdentifierToSemaphoreMap;
```

Comment:
Should this be wrapped in `#if TARGET_OS_WATCH`?

Reply:
Even though those two variable does not initialize in other platforms, wrapped in` #if TARGET_OS_WATCH` should be good.

Reply:
LGTM